core/src/cats/ddl/creates/postgresql.sql

```diff
@@ -485,4 +485,118 @@ INSERT INTO Status (JobStatus,JobStatusLong,Severity) VALUES
 --   but prevents errors if create script is called multiple times
 DELETE FROM Version WHERE VersionId<=2210;
 INSERT INTO Version (VersionId) VALUES (2210);
+-- subscription status part
+DROP VIEW IF EXISTS latest_full_size_categorized;
```

Comment:
This should be 
DROP VIEW IF EXISTS latest_full_size_categorized CASCADE;
So we cleanup depending object with.

Reply:
done